airmozilla/manage/templates/manage/survey_edit.html

```diff
@@ -40,58 +29,24 @@
     {% include 'manage/_default_form.html' %}
 
     <h3>Questions</h3>
+    <a class="survey-links" href="{{ url('manage:survey_questions', survey.id) }}">Edit questions here</a>
```

Comment:
The class name `survey-links` is not a great name. A better name would be `survey-link`. It's the same with the class names in `models.py`. You call it `class Question` instead of `class Questions` even though you know there might/will be more than one of them.